docs/pages/includes/add-role-to-user.mdx

```diff
@@ -91,6 +91,12 @@ Apply your changes:
 $ tctl create -f saml.yaml
 ```
 
+<Admonition type="warning">
+Note the `--with-secrets` flag in the `tctl get` command. This adds the value of
+`spec.signing_key_pair.private_key`. to `saml.yaml`. This is a sensitive value,
+so take precautions when creating this file and remove it after updating the resource.
```

Comment:
This is possibly implied, but just to triple-check: does "it" refer to the file or the value?

Reply:
The file.

Reply:
Setting script file permissions makes sense in a trusted environment. Removing the file makes sense in others, but doesn't work in a compromised environment in the first place. By being less specific we're leaving it to the reader to determine what security measures are appropriate for their environment. By saying what precautions are acceptable we open ourselves up to liability.